pilot/helpers/Project.py

```diff
@@ -560,7 +561,10 @@ def restore_files(self, development_step_id):
 
         clear_directory(self.root_path, ignore=self.files)
         for file_snapshot in file_snapshots:
-            update_file(file_snapshot.file.full_path, file_snapshot.content, project=self)
+            try:
```

Comment:
This can happen if/when people delete, rename or move the project directory, this at least makes us not crash in that case.